elements/pfe-content-set/demo/index.html

```diff
@@ -36,6 +37,7 @@ <h3>Container max: 500px</h3>
         <pfe-content-set-group>
           <h2 pfe-heading>Heading 1</h2>
           <p>Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore </p>
+          <pfe-cta priority="primary" color="base" on="dark"><a href="#">Primary</a></pfe-cta>
```

Comment:
@kylebuch8 Just so you know, `on="dark"` no longer needs to be manually added to CTA as the container element will add that when it's appropriate.

Reply:
I updated this and prettified/clarified the template file.  I was updating to add a few other variations to ensure there were no regressions.